app/controllers/admin/stripe_connect_settings_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Admin
+  class StripeConnectSettingsController < Spree::Admin::BaseController
+    before_filter :load_settings, only: [:edit]
+
+    def edit
+      return @stripe_account = { status: :empty_api_key } if Stripe.api_key.blank?
+      attrs = %i[id business_name charges_enabled]
+      @obfuscated_secret_key = obfuscated_secret_key
+      @stripe_account = Stripe::Account.retrieve.to_hash.slice(*attrs).merge(status: :ok)
+    rescue Stripe::AuthenticationError
+      @stripe_account = { status: :auth_fail }
+    end
+
+    def update
+      Spree::Config.set(params[:settings])
```

Comment:
Can you please explain and document this?
What do you expect in `params[:settings]`?
Is `Spree::Config.set()` really needed? It will get cached by Rails and affect other requests (and specs).

Reply:
Updating the cached config is exactly what we're trying to achieve here. This controller action is used by super admin users to update the settings the app is using.

Reply:
Ah OK, could you please specify that in the method documentation? Thanks!

Reply:
Resolved